pom.xml

```diff
@@ -30,22 +30,33 @@
     </scm>
 
     <properties>
-        <ddf.support.version>2.3.15</ddf.support.version>
+        <ddf.support.version>2.3.16</ddf.support.version>
         <maven-checkstyle-plugin.version>3.0.0</maven-checkstyle-plugin.version>
         <maven-jaxb2-plugin.version>0.14.0</maven-jaxb2-plugin.version>
         <maven-compiler-plugin.version>3.8.0</maven-compiler-plugin.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <gml-v_3_1_1-schema.version>1.1.0</gml-v_3_1_1-schema.version>
         <ows-v_1_0_0-schema.version>1.1.0</ows-v_1_0_0-schema.version>
-        <jaxb2.version>1.11.1</jaxb2.version>
         <maven-source-plugin.version>3.0.1</maven-source-plugin.version>
 
+        <!-- Neither of the below modules use semantic versioning correctly -->
+        <!-- Even though it's only a minor version bump <jaxb>2.3.0</jaxb> is a breaking change -->
+        <!-- and cannot be updated until our schemas use a newer version of JAXB2 -->
+        <!-- <jaxb2>0.11.0</jaxb2> also uses 2.3.0 -->
+        <jaxb.version>2.2.11</jaxb.version>
+        <jaxb2.version>0.10.0</jaxb2.version>
```

Comment:
❓ why did this change from 0.11.1 to 0.10.0 (yeah I know the 1.11.x stuff is wrong).

Reply:
Seems like we should use 0.11.1 or even 0.12.0 if possible.

Reply:
I don't actually know where 0.11.1 came from to be honest. It doesn't seem like it was being used at all in this repo. ddf-master is still using 0.9.4. @coyotesqrl might know.

Reply:
I do not recall where that version came from. I would definitely prefer the latest/greatest if possible.

Reply:
I just verified that we can't update with our current schemas. The newer versions of the `jaxb2-basics` plugin attach what I'll call method 2's to everything. (`toString2()`). Our schemas call methods that are no longer created with the correct parameters giving us a type mismatch
```
[ERROR]     method ToString.appendFields(ObjectLocator,StringBuilder,ToStringStrategy) is not applicable
[ERROR]       (argument mismatch; ToStringStrategy2 cannot be converted to ToStringStrategy)
```